cobra.go

```diff
@@ -40,14 +40,6 @@ var initializers []func()
 // Set this to true to enable it
 var EnablePrefixMatching bool = false
 
-// enables an information splash screen on Windows if the CLI is started from explorer.exe.
-var EnableWindowsMouseTrap bool = true
```

Comment:
I have no idea if anyone was using it, but this means you can't turn off the mousetrap/exit, right?